crates/fuel-core/src/graphql_api/service.rs

```diff
@@ -56,26 +60,24 @@ use std::{
         TcpListener,
     },
     pin::Pin,
-    sync::Arc,
 };
 use tokio_stream::StreamExt;
 use tower_http::{
     set_header::SetResponseHeaderLayer,
     trace::TraceLayer,
 };
 
```

Comment:
Could you connect imports please?=)

Reply:
Imports are as connected as they can be right now, `cargo fmt` should catch most of them too because we are on nightly